src/cryptoadvance/specter/templates/wallet/history/components/total_wallet_balances.jinja

```diff
@@ -1,113 +1,166 @@
+                            <th>
+                                <asset-label data-asset="{{asset}}" data-label="{{asset | assetlabel}}" edit-mode="enabled"></asset-label>
+                            </th>
+                        {% endfor %}
                     {% endif %}
-                    {% if specter.price_check %} 
-                        <span class="note" style="color: grey">({{ wallet.amount_immature | altunit }})</span>
-                    {% endif %}
-                    <tool-tip width="200px">
-                        <h4 slot="title">{{ _("What are immature outputs?") }}</h4>
-                        <span slot="paragraph">
-                        {{ _('The UTXO of coinbase transactions (mining rewards) have the special condition that they cannot be spent for at least 100 blocks. Their amount is not included in the total balance here.') }}
-                        </span>
-                    </tool-tip>
-                </div>
-            {% endif %}
+                {% endif %}
+            </table>
         </div>
-        {% if specter.is_liquid %}
-        {% if wallet.balance.get("assets", {}) %}
-            <div style="margin-bottom: 15px;">
-                <small style="line-height:30px">Assets:<br>
-                {% for asset in wallet.balance.get("assets",{}).keys() | sort %}
-                    {% set balance = wallet.balance.get("assets",{}).get(asset, {}) %}
-                    <div style="margin: 0 10px; display: inline-block;">
-                        {{ (balance.get("trusted", 0) + balance.get("untrusted_pending", 0) + balance.get("immature", 0)) | btcamount }}
-                        <asset-label data-asset="{{asset}}" data-label="{{asset | assetlabel}}" edit-mode="enabled"></asset-label>
-                    </div>
-                {% endfor %}
-                </small>
-            </div>
-        {% endif %}
-        {% include 'includes/overlay/liquid_assets_registry.html' %}
-        <small style="line-height: 30px;">
-            <button type="button" class="btn" style="margin: auto;" onclick="showPageOverlay('liquid-assets-registry')">{{ _("Show assets list") }}</button>
-        </small>
-        <br>
-        {% endif %}
     </div>
-    {% if wallet.rescan_progress or specter.utxorescanwallet == wallet.alias %}
+    {% if rescan_progress or specter.utxorescanwallet == alias %}
         <span id="wallet_rescan_data" style="display: contents;">
-            <h2>Rescanning blockchain: <span id="{{ wallet.alias }}_balances_wallet_rescan_percents">{{ "%.2f"|format(specter.info["utxorescan"] if specter.utxorescanwallet == wallet.alias else wallet.rescan_progress * 100) }}</span>%</h2>
+            <h2>Rescanning blockchain: <span id="{{ alias }}_balances_wallet_rescan_percents">{{ "%.2f"|format(specter.info["utxorescan"] if specter.utxorescanwallet == alias else rescan_progress * 100) }}</span>%</h2>
```

Comment:
I don't get this (also in the code beforehand): Shouldn't it be the other way around? Show the lengthier rescan progress if a full rescan has been launched and if not show the progress of the quicker UTXO rescan? Otherwise, it might be confusing to users who think they are already done when looking at the UTXO scan.
So, in pseudo-code: 
if rescan_progress then show rescan_progress * 100 else show specter.info["utxorescan"]

Reply:
Or why not just have to separate h2-tags for the complete rescan and the UTXO scan only?

Reply:
good question.

> Or why not just have to separate h2-tags for the complete rescan and the UTXO scan only?

Yeah.... would makes things much clearer.

Reply:
OK, I'll change that then.